.github/workflows/scylladb.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
   test:
     runs-on: ubuntu-latest-8-cores
-    timeout-minutes: 40
+    timeout-minutes: 50
```

Comment:
(Did this change make it slower, or is it just generally flaky because it takes ~40 minutes?)

Reply:
Generally flaky.